src/views/Unsubscribe/index.tsx

```diff
@@ -0,0 +1,177 @@
+import {
+    useCallback,
+    useState,
+} from 'react';
+import {
+    useNavigate,
+    useParams,
+} from 'react-router-dom';
+import {
+    gql,
+    useMutation,
+    useQuery,
+} from '@apollo/client';
+import {
+    ConfirmButton,
+    Message,
+} from '@ifrc-go/ui';
+import { useTranslation } from '@ifrc-go/ui/hooks';
+
+import Page from '#components/Page';
+import {
+    AlertSubscriptionsUnsubscribeQuery,
+    AlertSubscriptionsUnsubscribeQueryVariables,
+    UnsubscriptionMutation,
+    UnsubscriptionMutationVariables,
+} from '#generated/types/graphql';
+import useAlert from '#hooks/useAlert';
+
+import i18n from './i18n.json';
+import styles from './styles.module.css';
+
+const ALERT_SUBSCRIPTIONS = gql`
+    query AlertSubscriptionsUnsubscribe {
+        private {
+            id
+            userAlertSubscriptions {
+                items {
+                    id
+                    name
+                    isActive
+                    filterAlertAdmin1s
+                    filterAlertCountryId
+                }
+            }
+        }
+    }
+`;
+
```

Comment:
Why are we querying the subscriptions? Can't we use the subscription data fetched previously and pass that as a prop ?